src/utils/hooks/usePagination/usePagination.ts

```diff
@@ -1,11 +1,9 @@
 import { useState } from 'react';
 
-import { paginationInitialState, PaginationState } from '@virtualmachines/utils';
+import { paginationInitialState } from './utils/constants';
+import { PaginationState, UsePaginationValues } from './utils/types';
 
-type UsePagination = () => [
-  pagination: PaginationState,
-  onPaginationChange: (newPagination: PaginationState) => void,
-];
+type UsePagination = () => UsePaginationValues;
```

Comment:
u can move UsePagination to tils/types instead of importing usepaginationvalues and declaring usepagination here.. just import usepagination